sql/src/test/java/io/crate/operation/scalar/HashFunctionsTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.operation.scalar;
+
+import io.crate.analyze.symbol.Literal;
+import io.crate.types.DataTypes;
+import org.junit.Test;
+
+import static io.crate.testing.SymbolMatchers.isFunction;
+
+public class HashFunctionsTest extends AbstractScalarFunctionsTest {
+
+    @Test
+    public void testChecksums() throws Exception {
+        assertEvaluate("md5('©rate')", "53cee571b9fbab07cc894d55988cc70b");
```

Comment:
Assuming that `assertEvaluate` does a normalize (it does), this should be fine. Still better to test normalize explicitly. I added a test case.